src/app/components/elements/PromoContent.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { IsaacPodDTO } from "../../../IsaacApiTypes";
+import { apiHelper } from "../../services";
+import { Col, Row } from "reactstrap";
+import { IsaacContentValueOrChildren } from "../content/IsaacContentValueOrChildren";
+import { Tabs } from "./Tabs";
+
+export const PromoContent = ({ item }: { item: IsaacPodDTO }) => {
+  const { title, subtitle, value, image, url } = item;
+
+  const path = image && image.src && apiHelper.determineImageUrl(image.src);
+  return (
+    <Tabs tabContentClass="mt-3 mt-md-5" activeTabOverride={1}>
+      {{
+        "I Belong in Computer Science": (
+          <Row className="d-flex align-items-center justify-content-center">
+            <Col
+              xs={12}
+              md={6}
+              lg={5}
+              xl={4}
+              className="d-flex align-items-center justify-content-center"
+            >
+              <img
+                src={path}
```

Comment:
`path` can potentially be undefined. What would it happen in that case? Wouldn't we see a broken image in that case?

Reply:
path **shouldn't** be undefined, because when creating a newsPod in the content editor it adds an image for you. However just to be extra safe, in case someone decides to delete the image from the content for some reason, I've popped a default image in here.